dev/pages/Modal.vue

```diff
@@ -27,7 +27,7 @@ watch(showModal, (showing) => modalShowing.value = showing)
 
     <token>
       <w-modal title="Hello Warp!" :style="demoStyles" :left="showLeft" right @dismiss="showModal = false" v-model="showModal" @right="showModal = false">
-        <div class="space-x-8">
+        <div class="space-x-8 pt-4">
```

Comment:
This is just for visuals when we test component.

Before|After
---|---
<img width="674" alt="image" src="https://github.com/warp-ds/vue/assets/37986637/251d1c21-140c-41cb-b1a6-72af5a404df6">|<img width="666" alt="image" src="https://github.com/warp-ds/vue/assets/37986637/d5eb027c-c74f-4016-87cb-4308460d0f37">